### PR TITLE
implementing cd builtin

### PR DESCRIPTION
**Pull request by imenyoo2**

### CODE REVIEW

File: built-ins.c [hunk 1]
```diff
@@ -30,7 +30,51 @@ int handle_builtins(char **args, char **env)
...
 
+/**
+ * handleCD - handles the execution of cd command
+ * @args: array of cd args
+ * @env: environment array
+ * Return: void
+ */
+void handleCD(char **args, char **env)
+{
+	if (args[0] == NULL)
```
Comment (TaTsuuYa) [inline built-ins.c:50]: if the first arg is `NULL` shouldn't you just print an error and return

File: built-ins.c [hunk 2]
```diff
@@ -30,7 +30,51 @@ int handle_builtins(char **args, char **env)
...
+	{
+		if (chdir(getVarValue("$HOME", env)))
+		{
+			/* TODO: handle the errno set by chdir */
+		}
+	}
+	else if (args[1] != NULL)
```
Comment (TaTsuuYa) [inline built-ins.c:57]: and then if there's a second arg write the appropriate error and return

File: built-ins.c [hunk 3]
```diff
@@ -30,7 +30,51 @@ int handle_builtins(char **args, char **env)
...
+	{
+		/* TODO: this err msg could be not exactly correct */
+		write_std(getVarValue("$_", env), STDERR_FILENO);
+		write_std(": cd: too many arguments\n", STDERR_FILENO);
+	}
+	else
```
Comment (TaTsuuYa) [inline built-ins.c:63]: lately run the `chdir` and if it returns -1, print the error

File: built-ins.c (hunk 1 above)
Comment (imenyoo2) [reply built-ins.c:50]: no that means `ls` with no args has been supplied

Comment (TaTsuuYa) [reply built-ins.c:50]: which means that you have to write an error and quit the function with an empty return.
you mean `cd` right? not `ls`.

Comment (imenyoo2) [reply built-ins.c:50]: no `cd` is a short cut for `cd $HOME`

**Pull request merged**